packages/augur-artifacts/src/index.ts

```diff
@@ -58,13 +58,23 @@ export interface ContractAddresses {
     TimeControlled?: string;
     GnosisSafe?: string;
     ProxyFactory?: string;
-    ZeroXTrade?: string;
-    ZeroXExchange?: string;
     BuyParticipationTokens?: string;
     RedeemStake?: string;
     CashFaucet?: string;
     GnosisSafeRegistry?: string;
     HotLoading?: string;
+    ZeroXTrade?: string; // TODO sup with this?
```

Comment:
Whats the comment here asking?

Reply:
heheh I need to do a TODO sweep... At the time I wasn't sure if ZeroXTrade was still needed once we had the official 0x contracts.

Reply:
That looks like only TODO.